smdebug/core/utils.py

```diff
@@ -61,19 +61,11 @@ class FRAMEWORK(Enum):
 
 
 try:
-    import horovod.torch as hvd
-
-    # This redundant import is necessary because horovod does not raise an ImportError if the library is not present
-    import torch  # noqa
+    import horovod.tensorflow as hvd
 
     _hvd_imported = hvd
 except (ModuleNotFoundError, ImportError):
-    try:
-        import horovod.tensorflow as hvd
-
-        _hvd_imported = hvd
-    except (ModuleNotFoundError, ImportError):
-        _hvd_imported = None
+    _hvd_imported = None
```

Comment:
Can we also add a unknown error here? like what we did for smp import

Reply:
sure